include/bluetooth/gatt_pool.h

```diff
@@ -106,7 +106,7 @@ extern "C" {
  *
  *  @param _gp GATT service object with dynamic attribute allocation.
  *  @param _ccc CCC descriptor configuration.
- *  @param _cb CCC descriptor callback.
+ *  @param _ccc_changed CCC configuration changed callback.
```

Comment:
CCC is "Client Characteristic Configuration" so I would change to : "CCC value changed callback" or "CCC changed callback"

Reply:
Done.